apps/prairielearn/src/middlewares/selectAndAuthzInstanceQuestion.js

```diff
@@ -59,6 +59,10 @@ const middleware = asyncHandler(async (req, res, next) => {
       }
     }
   }
+}
+
+const middleware = asyncHandler(async (req, res, next) => {
```

Comment:
This doesn't seem be exported and it looks different to all the other handlers?

Reply:
It is exported on line 69, but I agree that this should be made consistent with the other files (that is, we should use `export default asyncHandler`. Will change.

Reply:
Changed in 48dcfa1.

Reply:
Thanks!